Sources/Transaction/Memos/DestinationMemo.swift

```diff
@@ -29,7 +29,10 @@ struct RecoverableDestinationMemo {
 
     func recover() -> DestinationMemo? {
         guard
-            DestinationMemoUtils.isValid(txOutPublicKey: txOutPublicKey, txOutTargetKey: txOutTargetKey, accountKey: accountKey),
+            DestinationMemoUtils.isValid(
+                txOutPublicKey: txOutPublicKey,
+                txOutTargetKey: txOutTargetKey,
+                accountKey: accountKey),
```

Comment:
You typically use double-indentation for parameter lists in actual calls:

```suggestion
                    txOutPublicKey: txOutPublicKey,
                    txOutTargetKey: txOutTargetKey,
                    accountKey: accountKey),

```

Reply:
single indentation is the general rule